packages/theme/src/cli/services/info.ts

```diff
@@ -84,3 +84,42 @@ function tabularSection(title: string, data: InlineToken[][]): AlertCustomSectio
     body: {tabularData: data, firstColumnSubdued: true},
   }
 }
+
+export async function formatThemeInfo(output: ThemeInfo, flags: {environment?: string}) {
+  const tabularData = Object.entries(output.theme).map(([key, val]) => {
+    if (key === 'editor_url' || key === 'preview_url') {
+      const url = String(val)
+      // Here, we create descriptive labels for the links
+      const label = key === 'editor_url' ? 'Open in Theme Editor' : 'Preview Theme'
+      return [formatKey(key), {link: {url, label}}]
+    } else if (key === 'id') {
+      return [formatKey(key), `#${val}`]
+    } else {
+      return [formatKey(key), `${val}`]
+    }
+  })
+
+  return {
+    customSections: [
+      ...(flags.environment
+        ? [
+            {
+              title: `Theme information`,
+              body: [{subdued: `Environment name: ${flags.environment}`}],
```

Comment:
I don't think it is ideal to show the environment name under the theme information because the environment is not actually related to any theme.

![image](https://github.com/user-attachments/assets/122978a6-4a72-4246-b416-0a5b2589d155)

I believe we could instead always display the environment name (when available) and present it in a separate section from the theme.

We may apply that enhancement in a different PR tho 🙇